src/Template/Element/sentences/audio.ctp

```diff
@@ -96,6 +98,14 @@ $this->AngularTemplate->addTemplate(
 
         <div class="audio-details" layout="column">
             <div class="license"><?= format(__('License: {license}'), ['license' => $licenseTemplate]) ?></div>
+            <div class="timestamp">
+                <div><?= format(__('Added')) ?></div>
+                <div class="since"><?= format(__('{date}'), [ 'date' => '{{ audio.created_ago }}' ]) ?></div>
+            </div>
+            <div class="timestamp">
+                <div><?= format(__('Last modified')) ?></div>
+                <div class="since"><?= format(__('{date}'), [ 'date' => '{{ audio.modified_ago }}' ]) ?></div>
+            </div>
```

Comment:
I’d like to suggest not displaying the "Last modified" section at all when the "last modified" date is the same as the "Added" one. That would reduce redundancy as I guess these dates are the same for most audio recordings.